Non-Developer.bundle

```diff
@@ -1,5 +1,5 @@
 #######################################
-# Non-Developer file for host non-development machine
+# Non-Developer file for host non-development machine #
```

Comment:
Same as previous heading. If not consistent, let's just simplify it, please.

Reply:
@hkdobrev Sure.

Reply:
```suggestion
# Non-Developer file for host non-development machine
```